ssl_checker.py

```diff
@@ -368,12 +403,36 @@ def get_args(self, json_args={}):
                             help='Show this help message and exit')
 
         args = parser.parse_args()
-
+        
```

Comment:
Let's skip the database integration, for now, it can be added later. At the moment, using a list of hosts is enough.